### PR TITLE
Added new contracts to Portal

### PR DESCRIPTION
**Pull request by kev1n-peters**
Hello DefiLlama team! I noticed that the Portal token bridge was missing some chains and I added them in this PR. Portal does a good amount of volume on Optimism and Arbitrum especially.

### CODE REVIEW

File: src/adapters/portal/index.ts [hunk 1]
```diff
@@ -228,50 +204,6 @@ const portalNativeAndWrappedTransfersFromHashes = async (
   return results.filter((tx) => tx) as EventData[];
 };
 
-// checks deposits for Solana as receipient chain, withdrawals for Solana as source chain and returns only those txs
-// the 'wormhole chain id' for Solana is 1
-const processLogsForSolana = async (logs: EventData[], chain: Chain) => {
```
Comment (0xngmi) [inline src/adapters/portal/index.ts:233]: why remove this? because it was commented out?

Comment (kev1n-peters) [reply src/adapters/portal/index.ts:233]: Yes, I can add it back if you'd like. We'd also like to figure out how to add first class Solana support in the future.

Comment (kev1n-peters) [reply src/adapters/portal/index.ts]: Added the removed code back

**Comment by 0xngmi** [comment]
haven't tested it but lgtm so will merge

**Pull request merged**